namespace java thrift.test

struct myByteStruct {
  1: byte a
}
